publication-model/src/main/resources/publication-ontology.ttl

```diff
@@ -261,6 +261,11 @@ nva:DegreePhd a rdfs:Class ;
   rdfs:isDefinedBy nva: ;
   rdfs:subClassOf nva:Degree .
 
+nva:ArtisticDegreePhd a rdfs:Class ;
+  rdfs:comment "" ;
+  rdfs:isDefinedBy nva: ;
+  rdfs:subClassOf nva:DegreePhd .
+
```

Comment:
Not sure what to add in comment here, to differentiate between DegreePhd and ArtisticDegreePhd.